package.json

```diff
@@ -59,6 +60,7 @@
     "@storybook/addon-centered": "^4.0.7",
     "@storybook/addon-knobs": "^4.0.7",
     "@storybook/addon-options": "^4.0.7",
+    "storybook-addon-styled-component-theme": "^1.1.1",
```

Comment:
minor: should this be added alphabetically?

Reply:
yes, great spot. will fix.